benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json

```diff
@@ -17,6 +17,9 @@
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "reasonableAdjustmentsOutstanding", "Label": "Alternative letter format required", "DisplayOrder": 16},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "hasUnprocessedAudioVideoEvidence", "Label": "Does case have unprocessed audio/video evidence?", "DisplayOrder": 17},
   {"LiveFrom": "16/12/2021", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpDueDate", "Label": "FTA response due date", "DisplayOrder": 18},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 19},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingDate", "Label": "Hearing Date", "DisplayOrder": 19},
+  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingEpimsId", "Label": "Hearing Venue", "DisplayOrder": 20},
```

Comment:
DisplayOrder seems wrong here and throughout the rest of the additions to this file

Reply:
Will leave as this was tested this way and passed SIT.